src/components/ProjectsList/__tests__/ProjectsList.test.tsx

```diff
@@ -1,14 +1,15 @@
 import { ProjectsList } from '../ProjectsList';
 import { MemoryRouter } from 'react-router-dom';
 import { render, screen } from '@testing-library/react';
-import renderer from 'react-test-renderer';
+
+jest.mock('../../../services/projectService');
 
 describe('display ProjectList component', () => {
   let mockProjectsRequest: any;
   let mockProjects: any;
```

Comment:
In the future we could try to use more specific type and avoid `any`

Reply:
yup that make sense